routers/api/v1/repo/branch.go

```diff
@@ -150,11 +150,6 @@ func DeleteBranch(ctx *context.APIContext) {
 		}
 	}
 
-	if ctx.Repo.Repository.IsMirror {
-		ctx.Error(http.StatusForbidden, "IsMirrored", fmt.Errorf("can not delete branch of an mirror repository"))
-		return
-	}
-
```

Comment:
Related or unrelated change?

Reply:
It's an unrelated change. But it's a duplicated line as line 129.